src/js/api/api.js

```diff
@@ -177,11 +177,8 @@ define([
         };
 
         this.load = function (toLoad) {
-            var plugin = this.getPlugin('vast') || this.getPlugin('googima') || this.getPlugin('freewheel');
-            if (plugin) {
-                _controller._model.set('preInstreamState', 'instream-idle');
-                plugin.destroy();
-            }
+            _controller._model.set('preInstreamState', 'instream-idle');
```

Comment:
Instream state should not be handled here. Remove and reset in controller when instream is instantiated.